pages/data-design/start-using-inspector.mdx

```diff
@@ -15,23 +37,69 @@ After <Link passHref href="/workspace/inspector#configuring-sources"><a>setting
+Open the <Link passHref href="/workspace/inspector#Overview"><a>Inspector overview</a></Link>, browse the list or search for a specific event. Select an event and detailed information about the health of the given event will appear in the panel on the right.
 
-From there you would be able to analyze the problems with the given event, if any, change event description in the tracking plan or add it to the tracking plan if it is not there yet.
-
-<Link href="/explore-tracking-plan/issue-types-in-inspector" passHref><a>List of issue types Inspector can detect</a></Link>  
-<br/>
-<br/>
+If the event you are reviewing isn’t present in your tracking plan in Avo, you can read the event schema and add the event to your tracking plan and if it does exist in your tracking plan but has any discrepancies, you can correct them in Avo to align your tracking plan with your actual tracking.
 
 When you fix the issue in the client you will be able to see how the problem is gradually resolved as you clients update.
 
-Make sure to install the <Link passHref href="/workspace/inspector#slack-integration">Inspector Slack integration</Link> to be alerted as soon as new issues are detected.
+Make sure to install the <Link passHref href="/workspace/inspector#slack-integration"><a>Inspector Slack integration</a></Link> to be alerted as soon as new issues are detected.
+
+## How to send metadata to the Inspector?
+
+### Install Inspector in your CDP
+
+If you send your events through a CDP (Customer Data Platform) you can configure the CDP to route the event metadata to Avo Inspector by setting up an Avo Inspector destination.
+
+The main benefit of this solution is that it does not require work from a developer and can be done without changing your application code and releasing a new version. All setup is done on the CDP's side.
+
+Learn how to route event to Inspector from:
+
+- <Link passHref href="/workspace/connect-inspector-to-segment"><a>Segment</a></Link>
+- <Link passHref href="/workspace/connect-inspector-to-rudderstack"><a>RudderStack</a></Link>
+- <Link passHref href="/workspace/connect-inspector-to-posthog"><a>PostHog</a></Link>
+- <Link passHref href="/help/troubleshooting"><a>Contact us if you want to use another CDP</a></Link>
+
+### Install Inspector with an HTML tag
+
+You can add Avo Inspector to a web page with an HTML tag. This option works for the users of the Google Tag Manager and other tag managers.
+
+Find the tag snippet and instructions to send data <Link passHref href="/implementation/inspector/sdk/web#installation-option-2-html-tag"><a>here</a></Link>.
+
+### Install Inspector SDK
+
+If you’re not using one of the supported CDPs you can send events data with an SDK. It will require one time setup and new app release and takes 2-30 minutes, depending on the developer’s experience. We recommend asking a developer that is familiar with the existing event tracking setup to help with this setup.
```

Comment:
```suggestion
If you’re not using one of the supported CDPs you can send events data with an SDK. It will require one time setup and a new app release and takes 2-30 minutes, depending on the developer’s experience. We recommend asking a developer that is familiar with the existing event tracking setup to help with this setup.
```